<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::miro::MiroDeployDialog</class>
 <widget class="QDialog" name="gams::studio::miro::MiroDeployDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assembly and Deploy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="gams::studio::fs::FileSystemWidget" name="fsWidget">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Deploy Configuration</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Execution environment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="targetEnvBox">
            <item>
             <property name="text">
              <string>Multi user</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Local multi user (uses inputDir)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Single user</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="testBaseButton">
            <property name="text">
             <string>Test Deploy</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="errorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>324</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="deployButton">
     <property name="text">
      <string>Deploy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gams::studio::fs::FileSystemWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">filesystemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>targetEnvBox</tabstop>
  <tabstop>testBaseButton</tabstop>
  <tabstop>deployButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
